.rubocop_todo.yml

```diff
@@ -0,0 +1,56 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2024-02-20 16:18:38 UTC using RuboCop version 1.57.2.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: TreatCommentsAsGroupSeparators, ConsiderPunctuation, Include.
+# Include: **/*.gemfile, **/Gemfile, **/gems.rb
+Bundler/OrderedGems:
+  Exclude:
+    - 'Gemfile'
+
+# Offense count: 2
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: final_newline, final_blank_line
+Layout/TrailingEmptyLines:
```

Comment:
Please fix these, because this PR introduces them. Seeing how often they show up you may need to change a setting on your editor.

Reply:
I'm voting for fixing all the findings in this case.
It's not that big change in the end.